corpora-api.yml

```diff
@@ -476,10 +480,7 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
-
```

Comment:
Please do not delete the spacing here. It helps with readability.

Reply:
Also likewise with lines 480 and 482.

Reply:
fixed